### ISSUE TITLE
Problem alexa not responding step 8

### ISSUE THREAD

**Issue by tonioecto** [alexa/alexa-avs-sample-app#575]
Hello,
I have followed all the instructions for setting avs on my raspberry pi and everything seem to work perfectly fine but when I want to talk to alexa it is not responding. I tried with both the wakeword and just the listen button, each time I hear the audio cue, I speak, I see the sound level moving so I guess the microphone is working and the I hear a second audio cue and then nothing, no answers. 
I have no error in all my windows, everything seems to be working exactly as it is supposed to and yet no answer from alexa. 

Any idea about what might be the problem ?

**Comment by Alexnagel** [comment]
Could be Alexa's volume is set low. Try setting it higher:
`Alexa, set volume to 10`

**Comment by danprime** [comment]
I, too am having the same problem and I tried @Alexnagel's suggestion of increasing the volume (no effect). I definitely see the sound bar moving and hear the second audio cue, there's some CPU activity but no log messages anywhere that I can see.

**Comment by tonioecto** [comment]
yes, the volume solution is not working for me either.

**Comment by elstryan** [comment]
Hi Folks, 
Can you please look in your application log and post a requestId from one of these requests? Under the javaclient directory there is a logs directory where you can find an application.log. Do a search for SpeechRecognizer and then just below a print out of the message you show find a line like this: 
`x-amzn-requestid: 0a60effffeb0f229-00003cd1-00030b6e-faee4a461d5ee16f-c5ed0379-1`

Additionally: after making a request can you try going to alexa.amazon.com and see if your utterance are visible there?

**Comment by danprime** [comment]
Thanks @elstryan 
x-amzn-requestid: 0a2dd1fffeb5a893-00004564-00070c04-5466c864e16720c7-bf7d54bd-11

Here's a link to the entire log if you would like to see: http://pastebin.com/vPz8JYCW

I found the history (alexa.amazon.com -> Settings ->History) there are a lot of entries that say "Text not available. Click to play recording" (see http://imgur.com/a/XJdt9 for screenshot)

When I click the Play recording, it sounds like my voice is playing at half or 1/4 speed (much slower) and has a lot of background noise.

I tested the volume of my microphone on a Windows Sound Recorder and the playback was clear and normal speed.
 
Seeing the history, perhaps the input/recording is consistently distorted that the voice recognition service can't understand it?

**Comment by elstryan** [comment]
Hi @danprime 

From checking the logs given that requestId it looks like the service can't parse the audio data. Could you possibly try a different microphone? Or try changing the speed/volume with which you speak into the microphone?

**Comment by hiephm** [comment]
I have exactly the same issue as @danprime. 

Seem that because the recording audio format is hard-coded to LPCM 1 channel, 16 bit, 16 kHz (https://github.com/alexa/alexa-avs-sample-app/blob/master/samples/javaclient/src/main/java/com/amazon/alexa/avs/AudioInputFormat.java#L52), so any different format will cause distortion.

Unfortunately my Windows machine only support 2 channels and 44.1kHz and above so I don't have any other option than to try it on a Linux machine or RPi.

I think this is a very common issue and should be listed in https://github.com/alexa/alexa-avs-sample-app/wiki/Troubleshooting, so that other developers won't waste their time debugging.

**Comment by liutao00** [comment]
@hiephm  I have the same problems on win7.  Do you succeed on linux or RPI? Anyone can run the app on win7?

**Comment by hiephm** [comment]
@liutao00 Yes, I've successed to use AVS on my VirtualBox running Ubuntu. In short, don't try it on Windows, Linux should works well.

**Comment by tweaty100** [comment]
hello @hiephm 
i am working on Linux (debian) but audio is not processing.when i click the listen button and say like "hello world" or anything i am not able to hear it back. what might be the problem .how can i correct this please help as soon as possible i am stuck in this .

thanks in advance

**Comment by VinnieSims** [comment]
Having this issue now. I've been running my AlexaPi for over 3-4 months now without any issues until now. Any fix?
